### PR TITLE
[API] Organization Service

### PR DESCRIPTION
**Pull request by dmitriyslaym**
https://github.com/capsulajs/capsulahub/issues/65

### CODE REVIEW

File: packages/service-organization/src/api/AddApiKeyRequest.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Claims } from './Claims';
+import { BaseOrganizationRequest } from './BaseOrganizationRequest';
+
+export interface AddApiKeyRequest extends BaseOrganizationRequest {
```
Comment (MikeSquall) [inline packages/service-organization/src/api/AddApiKeyRequest.ts]: What about `token` ?

File: packages/service-organization/src/api/Provider.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export type Provider = 'http' | 'websocket';
```
Comment (MikeSquall) [inline packages/service-organization/src/api/Provider.ts]: Why not supporting directly `RSocket` as the service already supports it ?

File: packages/service-organization/src/api/AddApiKeyRequest.ts (hunk 1 above)
Comment (dmitriyslaym) [reply packages/service-organization/src/api/AddApiKeyRequest.ts]: token will be provided as an argument of constructor and will be internally used in all requests

File: packages/service-organization/src/api/Provider.ts (hunk 2 above)
Comment (dmitriyslaym) [reply packages/service-organization/src/api/Provider.ts]: in the future we will do it, but right now we have transport providers only for http and websocket (https://github.com/capsulajs/capsulajs-transport-providers)

**Pull request merged**